rct229/rulesets/ashrae9012019/data/ashrae_90_1_prm_2019.motors.json

```diff
@@ -155,8 +155,8 @@
       "number_of_poles": "Any",
       "type": "Any",
       "synchronous_speed": "Any",
-      "minimum_capacity": 75.0,
-      "maximum_capacity": 99.999,
+      "minimum_shaft_input_power": 75.0,
+      "maximum_shaft_input_power": 99.999,
```

Comment:
I do not recommend to do such changes in the data file.
The data file is exported from OSSTD library. The process will need to go through their review first before the changes. Otherwise, it causes a lot unnecessary duplicating work in the future.
I suggest for now, put a map in the data function.

Reply:
Good point! Addressed.